CODEOWNERS

```diff
@@ -45,7 +45,7 @@
  
 /Workbooks/SapMonitor/** @microsoft/azuremonitorforsapsolutions @microsoft/applicationinsights-devtools
 
-/Workbooks/Virtual*Machine*/** @andrewki-msft @nasundar @rakshith210 @microsoft/applicationinsights-devtools
+/Workbooks/Virtual*Machine*/** @andrewki-msft @nasundar @rakshith210 @thaitranms @microsoft/applicationinsights-devtools
```

Comment:
Hey Andrew, could you make a Github team and add the team as a codeowner instead of individual users? That way it's easier to maintain PRs signoff and adding/removing users